bin/oneoffs/cas_statistic.py

```diff
@@ -0,0 +1,62 @@
+import os
+import pymongo
+from collections import Counter
+
+db_uri = os.getenv('SCITRAN_PERSISTENT_DB_URI', 'localhost:9001')
+db = pymongo.MongoClient(db_uri).get_database('scitran')
```

Comment:
You can also connect to the db by importing `api.config` and using `config.db`.

Example: https://github.com/scitran/core/blob/master/bin/oneoffs/timezone_shift.py#L7